tests/system/test_fim/test_synchronization/test_synchronization.py

```diff
@@ -136,6 +135,8 @@ def test_synchronization(folder_path, case, host):
         host_manager.run_command('wazuh-agent1', f'rm -rf {folder_path}')
         folder_path = f"'/{folder_path}/{folder_path}.txt'"
         query = " select * from fim_entry where full_path='\"{}\"'".format(folder_path)
+        agent_info = host_manager.run_command(host, '/var/ossec/bin/manage_agents -l')
+        agent_id = re.search(r'ID: (\d+)', agent_info).group(1)
```

Comment:
There are already functions that search for the agent ID, let's try using these.